Makefile.common

```diff
@@ -441,3 +441,5 @@ $(OCAMLRUN) $(ROOTDIR)/tools/stripdebug $(1) $(1).tmp \
 && $(INSTALL_PROG) $(1).tmp $(2) \
 && rm $(1).tmp
 endef # INSTALL_STRIPPED_BYTE_PROG
+
+HEADER_NAME = runtime-launch-info
```

Comment:
You were mentioning the possibility to add a brief comment to explain the role of `runtime-launch-info` and how it is bootstrapped. I think here would be a good place to put it.

Reply:
Aha - yes! I have pushed some extra text there.

> I want to approve this PR, I just have one last, small incomprehension: shouldn’t the bootstrap commit add a new file `boot/runtime-launch-info`?

I've clarified that in the comment, but no - `boot/runtime-launch-info` is the part which _can't_ be committed (the platform- and configuration-specific information).

Reply:
Thanks for the clarification.